cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsLaunchService.java

```diff
@@ -256,32 +254,59 @@ private void updateCloudformationWithLoadBalancers(AuthenticatedContext ac, Clou
         }
     }
 
-    private AwsLoadBalancerScheme determinePublicVsPrivateSchema(AwsNetworkView awsNetworkView, AmazonEC2Client amazonEC2Client) {
-        String subnetId = awsNetworkView.getExistingSubnet();
-        String vpcId = awsNetworkView.getExistingVpc();
+    private void convertLoadBalancer(CloudLoadBalancer cloudLoadBalancer, List<CloudResource> instances,
```

Comment:
It converting from `CloudLoadBalancer` to `AwsLoadBalancer`. So in that sense it was always creating an instance of `AwsLoadBalancer` and setting its properties based on the properties of `CloudLoadBalancer`. That piece of the logic hasn't really changed. I will update it to return the `AwsLoadBalancer` and add it to the list in the parent method instead of here.